src/Entity/CreationLog.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Entity;
+
+use App\Book;
+use DateTime;
+use DateTimeZone;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * The CreationLog is the log of exports.
+ * @ORM\Entity(repositoryClass="App\Repository\CreationLogRepository")
+ * @ORM\Table(
+ *     name="books_generated",
+ *     indexes={
+ *         @ORM\Index(name="time", columns={"time"}),
+ *         @ORM\Index(name="lang", columns={"lang"})
+ *     }
+ * )
+ */
+class CreationLog {
```

Comment:
Entities are usually singular names of the records inside them. Maybe `GeneratedBook` (to be similar to the table name)?

Reply:
:+1: I wanted to change it but wasn't sure if it'd be too different than what people are used to. `GeneratedBook` sounds good!

Reply:
FUERDAI IS HERE FROM WIKIPEDIA AND WILL VANDALISE GITHUB ON WHEELS